packages/sources/polygon-test/src/endpoint/router.ts

```diff
@@ -0,0 +1,35 @@
+import { RoutingTransport } from '@chainlink/external-adapter-framework/transports/meta'
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { customSettings } from '../config'
+import { ProviderResponseBody, httpTransport, inputParameters } from './tickers'
+
+export interface RequestParams {
+  base: string
+  quote: string
+}
+
+export type EndpointTypes = {
+  Request: {
+    Params: RequestParams
+  }
+  Response: SingleNumberResultResponse
+  CustomSettings: typeof customSettings
+  Provider: {
+    RequestBody: never
+    ResponseBody: ProviderResponseBody
+  }
+}
+
+export const routingTransport = new RoutingTransport<EndpointTypes>(
+  {
+    HTTP: httpTransport,
+  },
+  () => 'HTTP',
+)
```

Comment:
We can remove the routing transport and just use the http transport directly

Reply:
Done